test/frame_rendering_test.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
+import 'dart:async';
+import 'dart:io';
+
+import 'package:devtools/globals.dart';
+import 'package:devtools/service_extensions.dart' as extensions;
+import 'package:devtools/service_manager.dart';
+import 'package:devtools/timeline/frame_rendering.dart';
+import 'package:devtools/vm_service_wrapper.dart';
+import 'package:test/test.dart';
+
+import 'support/flutter_test_driver.dart';
+
+void main() {
+  group('frame rendering tests', () {
+    FlutterRunTestDriver _flutter;
+    VmServiceWrapper service;
+    FramesTracker framesTracker;
+
+    setUp(() async {
+      _flutter = FlutterRunTestDriver(Directory('test/fixtures/flutter_app'));
+
+      await _flutter.run(withDebugger: true);
+      service = _flutter.vmService;
+
+      setGlobal(ServiceConnectionManager, ServiceConnectionManager());
+
+      await serviceManager.vmServiceOpened(service, Completer().future);
+
+      framesTracker = FramesTracker(service);
+      framesTracker.start();
```

Comment:
starting the frames tracker seems like something to do in a test case rather than startup?